### ISSUE TITLE
Failed: FanOutThroughputSpec

### ISSUE THREAD

**Issue by johanandren** [akka/akka#28249]
https://jenkins.akka.io:8498/job/akka-artery-aeron-udp-remote-tests/208/consoleFull

```
must be great for size-10k, burstSize = 333, payloadSize = 10000 (on node 'node-3', class akka.remote.artery.FanOutThroughputSpecMultiJvmNode3) *** FAILED *** (5 seconds, 966 milliseconds)
```

Very curious because:

```
[error] [JVM-1] WARNING: EXCEPTION, please implement akka.remote.testconductor.ConductorHandler.exceptionCaught() for proper handling.
```

**Comment by chbatey** [comment]
https://jenkins.akka.io:8498/job/akka-artery-aeron-udp-remote-tests/224/consoleFull

**Comment by patriknw** [comment]
The first fail is something about the testing infrastructure so not much we can do: "Connection to a5.moxie closed by remote host".

The second failure is more interesting:

```
[info] [JVM-1] [ERROR] [12/12/2019 08:56:02.105] [FanOutThroughputSpec-akka.actor.internal-dispatcher-3] [akka://FanOutThroughputSpec/user/size-100-snd2] null
[info] [JVM-1] java.util.NoSuchElementException
[info] [JVM-1] 	at org.agrona.collections.Object2IntHashMap$AbstractIterator.findNext(Object2IntHashMap.java:820)
[info] [JVM-1] 	at org.agrona.collections.Object2IntHashMap$EntryIterator.next(Object2IntHashMap.java:928)
[info] [JVM-1] 	at org.agrona.collections.Object2IntHashMap$EntryIterator.next(Object2IntHashMap.java:922)
[info] [JVM-1] 	at scala.collection.convert.Wrappers$JIteratorWrapper.next(Wrappers.scala:44)
[info] [JVM-1] 	at scala.collection.Iterator$$anon$10.next(Iterator.scala:459)
[info] [JVM-1] 	at scala.collection.Iterator.foreach(Iterator.scala:941)
[info] [JVM-1] 	at scala.collection.Iterator.foreach$(Iterator.scala:941)
[info] [JVM-1] 	at scala.collection.AbstractIterator.foreach(Iterator.scala:1429)
[info] [JVM-1] 	at scala.collection.TraversableOnce.toMap(TraversableOnce.scala:321)
[info] [JVM-1] 	at scala.collection.TraversableOnce.toMap$(TraversableOnce.scala:319)
[info] [JVM-1] 	at scala.collection.AbstractIterator.toMap(Iterator.scala:1429)
[info] [JVM-1] 	at akka.remote.artery.compress.CompressionTable.dictionary(CompressionTable.scala:25)
[info] [JVM-1] 	at akka.remote.artery.MaxThroughputSpec$Sender$$anonfun$waitingForCompression$1.applyOrElse(MaxThroughputSpec.scala:194)
[info] [JVM-1] 	at akka.actor.Actor.aroundReceive(Actor.scala:533)
[info] [JVM-1] 	at akka.actor.Actor.aroundReceive$(Actor.scala:531)
[info] [JVM-1] 	at akka.remote.artery.MaxThroughputSpec$Sender.aroundReceive(MaxThroughputSpec.scala:155)
```

**Comment by patriknw** [comment]
I'm pretty sure this was fixed in https://github.com/akka/akka/pull/28363

The failed build was at 3c1c06fb88, which was before the fix in 0d7ad2794b.

```
* | | | 0d7ad2794b 2019-12-12 | fix concurrent access to CompressionTable Object2IntHashMap (#28363) [Patrik Nordwall]
* | | | ded2373941 2019-12-12 | [doc] `an` for a better flow (#28359) [Robert Stoll]
| |/ /
|/| |
* | | 3c1c06fb88 2019-12-11 | Fix a single grammar error in docs (#28352) [Michał Sieczkowski]
```
